modules/odc-editing-health-checks.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// applications/application-health
+
+[id="odc-editing-health-checks"]
+= Editing health checks using the Developer perspective
+
+You can use the *Topology* view to edit health checks added to your application, modify them, or add more health checks.
+
+.Prerequisites:
+* You have switched to the *Developer* perspective in the web console.
+* You have created and deployed an application on {product-title} using the *Developer* perspective.
+* You have added health checks to your application.
+
+.Procedure
+. In the *Topology* view, right-click your application and select *Edit Health Checks*. Alternatively, in the side panel, click the *Actions* drop-down list and select *Edit Health Checks*.
+. In the *Edit Health Checks* page:
+
+* To remove a previously added health probe, click the minus sign adjoining it.
+* To edit the parameters of an existing probe:
++
+.. Click the *Edit Probe* link next to a previously added probe to see the parameters for the probe.
```

Comment:
Edit Probe is near to the title of the Probe.

Reply:
Imho, the current description is correct and the UI is clear enough that the user will not get confused. So I will not make any change to that if you don't mind.